packages/fxa-auth-server/config/index.ts

```diff
@@ -1804,6 +1804,13 @@ conf.set('smtp.subscriptionSupportUrl', `${baseUri}/support`);
 
 conf.set('isProduction', conf.get('env') === 'prod');
 
+if (conf.get('env') === 'stage') {
```

Comment:
I think we can do this by adding something like [here](https://github.com/mozilla/fxa/blob/14b0c2960f517f45786ab2859cbfdc66f1799019/packages/fxa-auth-server/config/index.ts#L1730)

Use content-server config to get the `accounts.stage.firefox.com` part.

Reply:
Aha, just noticed [this whole setup](https://github.com/mozilla/fxa/blob/14b0c2960f517f45786ab2859cbfdc66f1799019/packages/fxa-auth-server/config/index.ts#L1734-L1760).